### ISSUE TITLE
Not able to input the boolean array as one parameter

### ISSUE THREAD

**Issue by yuxiaomin** [swagger-api/swagger-ui#2409]
This issue was found on swagger-ui 2.1.5 and on the latest 2.2.3 this issue could be reproduced.

Below is the json spec for the boolean array parameter

```
         {
            "name": "f_boolean",
            "in": "formData",
            "description": "",
            "required": false,
            "type": "array",
            "items": {
              "type": "boolean",
              "format": "",
              "default": false
            }
          }
```

In UI this parameter is rendered as following
![image](https://cloud.githubusercontent.com/assets/2513729/18598803/6d16579c-7c1a-11e6-8c4f-99b0bfdd1d66.png)

I am not able to input a boolean array. The boolean array parameter is treated as a boolean parameter.

**Comment by fehguy** [comment]
Please try setting `"default": "false"` in quotes just to see

**Comment by yuxiaomin** [comment]
@fehguy It did not make difference.

**Comment by yuxiaomin** [comment]
I found that if the type is boolean then the template templates.param_list will used to render the parameter view, thus the input is a multiple select not a textarea box.

**Comment by fehguy** [comment]
I'm confused.  Is that what you're expecting?

**Comment by yuxiaomin** [comment]
@fehguy Let me rephrase. I mean if the type of parameter is array and the item type is boolean I think the correct UI is showing a textarea box for user to enter multiple lines. Now it shows a select as the screenshot that I posted in my first comment. User can not enter the value for Array[boolean]. The issue in code is that the template to render the parameter view is wrong.

**Comment by webron** [comment]
Please try 3.X and reopen if needed.
